scripts/globals/pets/wyvern.lua

```diff
@@ -33,6 +33,220 @@ local wyvernTypes = {
     [dsp.job.RUN] = WYVERN_MULTI
 }
 
+-- It seems that the logic is:
+-- JOB ABILITIES
+    -- Includes physical job abilities with a duration, e.g. Berserk, Seigan, Meditate
+        -- i.e. Probably should include Cover
+    -- Excludes magical job abilities with a duration, e.g. Dark Arts, Light Arts
+        -- i.e. Probably should exclude Composure
+    -- Excludes on hit job abilities, e.g. Sneak Attack, Elemental Seal; EXCEPTION: Boost
+        -- i.e. Probably should exclude Barrage
+-- SPELLS
+    -- Includes battle spell buffs, e.g. stoneskin, spikes, regen, utsusemi, samba, minuet, rolls, sambas
+        -- i.e. Probably should include Migawari, Avatar's Favor
+    -- Excludes non-battle spell buffs, e.g. Monomi, Invisible
+        -- i.e. Probably should exclude Mazurka, Jigs
+-- CONSUMABLES
+    -- Includes attribute boosting medicines, e.g. Icarus Wing, Agility Potion, Giant's Drink
+    -- Excludes food, i.e. Instant Noodles, Meat Mithkabob
+local empathyEffects = {
```

Comment:
this table could be moved to a flag in src/map/status_effect.h along with the flag set in status_effect db table

Reply:
Yes, I was thinking about that. I didn't know if that was the best approach as it seems that new status effects that have been added might have been mis-flagged by contributors who weren't familiar with the flags, e.g. Celerity is flagged `EFFECTFLAG_DEATH` but may also need to be flagged `EFFECTFLAG_MAGIC_END` (but my Russian is rusty and I'm not sure I fully understand the flag).

Do you think it is worth it to make it a flag?

Reply:
update status_effects set flag = flag | <val> where effectid in(list, of, status, ids)